### DISCUSSION TITLE
i want to call action inside custom loop in view inside infolist

### DISCUSSION THREAD

**Discussion by mohamedAhmed97** [filamentphp/filament#10486] [Help]
in UserResource i call "ipwhois" for get active session of this user inside table and i want to add action for end active session 
i want to add specification for each action 
 
![Screenshot from 2023-12-25 11-11-38](https://github.com/filamentphp/filament/assets/58955899/27a6e691-89e9-455e-8855-e4ed000bf598)
![Screenshot from 2023-12-25 11-11-21](https://github.com/filamentphp/filament/assets/58955899/2e960431-3cf6-4714-849e-51c819140d96)

![Screenshot from 2023-12-25 11-10-03](https://github.com/filamentphp/filament/assets/58955899/3c5b42e1-22c4-4000-bf48-0a5d20a0852b)
![Screenshot from 2023-12-25 11-09-05](https://github.com/filamentphp/filament/assets/58955899/6ffc15b5-b6d7-4bb7-824b-982adc2f158c)

**Comment by StarterES-sgaleano** [comment]
hey @mohamedAhmed97 do you manage to get this working for you? Im having the same issue. In my case i have a custom Entry component with an array as state and i want to have an action for each im able to render the action but its not working.
Thanks in advance
